### DISCUSSION TITLE
Тяжелые доспехи охотника на драконов / ОНД

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#486] [General]
Я вот что подумал. в диалоге про тяжёлые доспехи ОНД герой говорит про улучшение доспехов. 
```D
instance DIA_Bennet_BetterArmor(C_Info)
{
	npc = SLD_809_Bennet;
	nr = 8;
	condition = DIA_Bennet_BetterArmor_Condition;
	information = DIA_Bennet_BetterArmor_Info;
	permanent = FALSE;
	description = "Я знаю, как можно еще улучшить доспехи.";
};


func int DIA_Bennet_BetterArmor_Condition()
{
	if((PlayergetsFinalDJGArmor == TRUE) && (hero.guild == GIL_DJG))
	{
		return TRUE;
	};
};

func void DIA_Bennet_BetterArmor_Info()
{
	AI_Output(other,self,"DIA_Bennet_BetterArmor_15_00");	//Я знаю, как можно еще улучшить доспехи.
	AI_Output(self,other,"DIA_Bennet_BetterArmor_06_01");	//(ухмыляется про себя) Ну, расскажи мне.
	AI_Output(other,self,"DIA_Bennet_BetterArmor_15_02");	//Можно покрыть драконьи чешуйки магической рудой.
	AI_Output(self,other,"DIA_Bennet_BetterArmor_06_03");	//(смеется) Эта мысль приходила и ко мне. Да, ты прав.
	AI_Output(self,other,"DIA_Bennet_BetterArmor_06_04");	//Мои новые доспехи превосходят все, что ты когда-либо видел. Они очень легкие и очень прочные.
	AI_Output(self,other,"DIA_Bennet_BetterArmor_06_05");	//Они СОВЕРШЕННЫ.
	AI_Output(self,other,"DIA_Bennet_BetterArmor_06_06");	//Ты можешь купить их, если хочешь. Я не предложил бы их абы кому, а цена только-только покрывает стоимость производства.
};
```

Может сделать дополнительную возможность если у героя есть средние доспехи ОНД, улучшить их до тяжелых доспехов ОНД? например за 10 000 -15 000 золотых:-)

**Comment by dosinabox** [comment]
То есть, делать скидку на тяжелые доспехи, если у ГГ уже есть средние, и при этом заменять средние на тяжелые при оплате?

**Reply by Kor-Angar** [reply]
хорошо что я потихоньку начал делать мини-моды. так как  мысли одни и теже приходят постоянно  и оттого в голове появляется де-жавю.

https://steamcommunity.com/sharedfiles/filedetails/?id=2853204144

**Comment by Kor-Angar** [comment]
так же  хочется напомнить ситуацию со средними доспехами, а именно разброс в цене у Яна и Беннета. по моему ты это так и не исправила.

**Comment by Kor-Angar** [comment]
ты цену у Яна на средние доспехи ОнД не правила? 

https://github.com/dosinabox/g2nr_unofficial_update/blob/aed56ebb175bc52740bf908b2ece4ef16490b17d/PrjGOTHIC/Story/Dialoge/DIA_DJG_714_Jan.d#L517

**Reply by Kor-Angar** [reply]
> нет, осталось 12000 как в оригинале

у Яна чешую в неоф. обновление и деньги отобрать же нельзя?

**Reply by Kor-Angar** [reply]
> нет, осталось 12000 как в оригинале


![image](https://user-images.githubusercontent.com/49514764/209464162-f6653c09-fa80-4ad6-9026-f8006c3f22bd.png)

**Comment by dosinabox** [comment]
> Может, имеет смысл слегка видоизменить доспехи охотников на драконов, которые носят NPC? Я имею в виду не модели, а только сами текстуры, сделав им этакий рескин: убрать украшения, заменить коричневую/рыжую отделку (позолоту?) стальным цветом. По крайней мере это решило бы логическую/сюжетную дыру плане "откуда куча бомженаёмников достала чертовски дорогие топовые доспехи, хотя у Беннета не было для этого драконьей чешуи?" (так как "драконов не видели уже многие века"). Герой в 3 главе тоже получал бы "дешёвый" вариант доспехов, как и все остальные, а вот средние и тяжёлые покупал бы именно классические, оригинальные (ибо при их изготовлении уже используется драконья чешуя). Параметры "подделок" и "подлинников" при этом пусть остаются одинаковыми, поскольку суть лишь в визуальном отличии во избежание геймплейно-сюжетной сегрегации.
Ну и в 5 главе можно переодеть Горна из поддельной тяжёлой в настоящую тяжёлую броню ОнД (охота окончена, чешуя на трупах драконов есть в избытке, горы золота в сокровищницах Безымянный почти не трогал, так что Горн без проблем мог раздобыть и притащить необходимые материалы и средства Беннету для приведения доспехов к более каноническому виду).

**Reply by Kor-Angar** [reply]
Сейчас в моделе имя текстуры написано в правильном формате? То есть поддерживает ли модель разные "шкурки"? Я по моему тебе скидывал свой архив моделей в котором была изменена "маска" имени. это к тому что если модель будет задействован одна и таже, и будет   общий прототип инстанции этого доспеха и в инстанции просто указывается разные шкурки. В таком случае разнообразие одного доспеха может быть  большим. И дальше уже вопрос вкуса, вплоть до того что номер текстуры можно брать чуть ли не из файла ини. Если же вопрос именно про то какая она должна быть, то я не знаю. Это уже такой вопрос ощущения. Одним нравится то, а другим это.

***

* видос https://youtu.be/prU1D-tdVM0

* моделька [Hum_DJG_Armor.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/10299733/Hum_DJG_Armor.zip)
* инстанция [ItAr_DJG.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/10299734/ItAr_DJG.zip)

***

прочие доспехи тут https://github.com/dosinabox/g2nr_unofficial_update/discussions/351
